pyvrp/read.py

```diff
@@ -21,12 +20,16 @@ def round_nearest(vals: np.ndarray):
     return np.round(vals).astype(int)
 
 
-def convert_to_int(vals: np.ndarray):
+def round_down(vals: np.ndarray):
     return vals.astype(int)
```

Comment:
```suggestion
def truncate(vals: np.ndarray):
    return vals.astype(int)
```
Maybe? Since we also call this function `"trunc"` in the `ROUND_FUNCS` options.